docs/cookbooks/studio/usecases-functions.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Function use cases
+sidebar_label: Function use cases
+---
+
+The following are some of the use cases of the **Functions** section in **Studio**.
+
+## Display  weather in your bot in a caraousel node
+
+1. [Add the weather API](https://docs.yellow.ai/docs/platform_concepts/studio/api/add-api#1-add-a-new-api-using-url) to the platform.
+2. [Create a flow ](https://docs.yellow.ai/docs/platform_concepts/studio/build/Flows/journeys#2-create-a-flow)and add an [API node](https://docs.yellow.ai/docs/platform_concepts/studio/build/nodes/action-nodes#21-api). Select the relevant API in this node to fetch weather data.
+
+![](https://i.imgur.com/Jklf0Ia.png)
+
+3. Save the response from the API node into a variable for further use.
+
+![](https://i.imgur.com/dKN4rrr.png)
+
+4. Navigate to **Functions** and pass the variable containing the API response and write a code that matches the dynamic format of the carousel node. Click [here](https://docs.yellow.ai/docs/platform_concepts/studio/build/nodes/message-nodes#4-carousel) for the dynamic format.
+
+![](https://i.imgur.com/Jklf0Ia.png)
+
+5. Include a [Function node](https://docs.yellow.ai/docs/platform_concepts/studio/build/nodes/action-nodes#24-function) and choose the function you created in the previous step. Save the response of this node into another variable.
+
+![](https://i.imgur.com/PQhxMKE.png)
+
+6. Add a [carousel node](https://docs.yellow.ai/docs/platform_concepts/studio/build/nodes/message-nodes#4-carousel) to the flow. In the **Fetch from** field of the carousel node, fetch the variable created in the previous step to populate the carousel with the desired data.
+
+![](https://i.imgur.com/Mlh4Xzv.png)
+
+:::note
+Instead of including the Function node, you can also pass the function in the **Parse API response** field.
+![](https://i.imgur.com/eIReCNg.png)
+:::
+
+
+## Calculate 45 days before the present date
```

Comment:
## Function (code) to manipulate dates

Add example as 45 days before the present date

Explain date object and say, you can write functions to calculate based on these fields ex. timestamp, year, date etc.

Reply:
seems like a stretch, we have taken a specific example to explain here. Better to stick only to that. Use case guides are intended towards a particular use case